db/scripts/seed_renewal_basis_for_policies_script.rb

```diff
@@ -1,11 +0,0 @@
-# frozen_string_literal: true
-
-# heroku run:detached --tail \
-#   rails runner db/scripts/seed_renewal_basis_for_policies_script.rb
-
-count = Policy.where(renewal_basis: nil)
```

Comment:
Dropping the whole `db/scripts` concept. Too complicated when it comes to self-hosting.